Update About This Release Page for SI Tooling section (#3869)
wso2/docs-apim: en/docs/get-started/about-this-release.md

### New features

- **Async API View**
    

> 	 This release introduces a new view - 'Async API View', in addition to the existing 'Source/Design View' and 'Wizard view'. 

Reviewer: ```suggestion
	 This release introduces a new 'Async API' view in addition to the existing 'Source/Design' views and 'Wizard' view. 
```

	 
	 Using the Async API View, you can generate an [Async API](https://www.asyncapi.com/) specification for a given Siddhi app. 
	 
	 The following sinks/sources can be exposed as an Async API : [websocket-server (sink)](https://siddhi-io.github.io/siddhi-io-websocket/api/3.0.2/#websocket-server-sink),  [web socket-server (source)](https://siddhi-io.github.io/siddhi-io-websocket/api/3.0.2/#websocket-server-source), [websocket (source)](https://siddhi-io.github.io/siddhi-io-websocket/api/3.0.2/#websocket-source), [sse-server (sink)](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink), [websubhub (source)](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source)
     

>      Refer example use case [here](https://apim.docs.wso2.com/en/4.0.0/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/).

Reviewer: ```suggestion
      See the [example use case](https://apim.docs.wso2.com/en/4.0.0/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/).
```

- **Tooling support for webhubsub Source and webhubsub Sink**
    

> 	 Tooling support has been added for the newly introduced [webhubsub source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source) and [webhubsub sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink). For feature description, refer to the Streaming Integrator new features section above.

Reviewer: ```suggestion
	 Tooling support is added for the newly introduced [webhubsub source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source) and [webhubsub sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink). For details, see the **Streaming Integrator - New Features** section above.
```

- **Tooling support for SSE Source and SSE-server Sink**

> 	 Tooling support has been added for the newly introduced [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source) and [SSE-server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink) . For feature description, refer to the Streaming Integrator new features section above.

Reviewer: ```suggestion
	 Tooling support is added for the newly introduced [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source) and [SSE-server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink). For details, see the **Streaming Integrator - New Features** section above.
```

	 
- **Siddhi Error Replay for 'Store' type errors**

> 	 This release enables error replay for database connection errors. Errors occurred during ADD, DELETE, UPDATE, ADDORUPDATE operations can be replayed from the error store explorer. Refer tutorial on handling store-type errors [here](https://apim.docs.wso2.com/en/4.0.0/use-cases/streaming-tutorials/handling-requests-with-errors/#step-7-handle-events-that-fail-at-database-level).

Reviewer: ```suggestion
	 This release enables **error replay** for database connection errors. Errors that occur during ADD, DELETE, UPDATE, ADDORUPDATE operations can be replayed from the Error Store explorer. 
	 
	 See the tutorial on [handling store-type errors](https://apim.docs.wso2.com/en/4.0.0/use-cases/streaming-tutorials/handling-requests-with-errors/#step-7-handle-events-that-fail-at-database-level).
```
